hooks/handleGlobalError.ts

```diff
@@ -4,21 +4,16 @@ import { Dispatch, MiddlewareAPI, PayloadAction, isRejectedWithValue } from '@re
 export const handleGlobalError = (api: MiddlewareAPI) => (next: Dispatch) => (action: PayloadAction<ErrorPayloadType>) => {
   if (isRejectedWithValue(action)) {
     const status = action.payload.status
+    const error: any = action.payload.data?.messages[0].message || action.payload.data.messages
```

Comment:
App still crashes when trying to connect to server. try to move error declaration in try block